lib/packages/core/lib/src/views/media/media_preview.dart

```diff
@@ -0,0 +1,280 @@
+                      _defImage(),
+                    )
+                  : Padding(
+                      padding: EdgeInsets.symmetric(
+                        vertical: 2.h,
+                      ),
+                      child: _screenBuilder.video(
+                        context,
+                        _defVideo(),
+                      ),
+                    ),
+            ),
+          ),
+          const Spacer(),
+          _defPreviewBar()
+        ],
+      );
+    } else if (mediaList.first.mediaType == LMChatMediaType.document) {
+      return Container();
+    }
+    return const SizedBox();
+  }
+
+  Container _defPreviewBar() {
+    return Container(
+      decoration: BoxDecoration(
+          color: LMChatTheme.theme.container,
+          border: Border(
+            top: BorderSide(
+              color: LMChatTheme.theme.disabledColor,
+              width: 0.1,
+            ),
+          )),
+      padding: EdgeInsets.only(
+        left: 2.w,
+        right: 5.0,
+        top: 2.h,
+        bottom: 4.h,
+      ),
+      child: SizedBox(
+        height: 15.w,
+        width: 100.w,
+        child: Center(
+          child: _defPreviewList(),
+        ),
+      ),
+    );
+  }
+
+  ListView _defPreviewList() {
+    return ListView.builder(
+      padding: EdgeInsets.zero,
+      itemCount: mediaList.length,
+      scrollDirection: Axis.horizontal,
+      itemBuilder: (context, index) => GestureDetector(
+        onTap: () {
+          currPosition = index;
+          rebuildCurr.value = !rebuildCurr.value;
+        },
+        child: Container(
+          margin: const EdgeInsets.symmetric(
+            horizontal: 3.0,
+          ),
+          clipBehavior: Clip.hardEdge,
+          decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(12.0),
+              border: currPosition == index
+                  ? Border.all(
+                      color: LMChatTheme.theme.secondaryColor,
+                      width: 2.0,
+                      strokeAlign: BorderSide.strokeAlignOutside,
+                    )
+                  : null),
+          width: 15.w,
+          height: 15.w,
+          child: mediaList[index].mediaType == LMChatMediaType.image
+              ? _defImageThumbnail(index)
+              : _defVideoThumbnail(index),
+        ),
+      ),
+    );
+  }
+
+  LMChatImage _defImage() {
+    return LMChatImage(
+      imageUrl: mediaList[currPosition].mediaUrl!,
+      style: const LMChatImageStyle(
+        boxFit: BoxFit.cover,
+      ),
+    );
+  }
+
+  ClipRRect _defImageThumbnail(int index) {
+    return ClipRRect(
+      borderRadius: BorderRadius.circular(8.0),
+      child: LMChatImage(
+        imageUrl: mediaList[index].thumbnailUrl ?? mediaList[index].mediaUrl,
+        style: const LMChatImageStyle(
+          boxFit: BoxFit.cover,
+        ),
+      ),
+    );
+  }
+
+  Container _defVideo() => Container();
+
+  Widget _defVideoThumbnail(int index) {
```

Comment:
customisation miss!

Reply:
Function not being used currently, acting as a placeholder